In [1]:
import pandas as pd
import numpy as np
import warnings
warnings.filterwarnings('ignore')

In [2]:
df=pd.read_csv('Q9_b.csv')
df.head()

Unnamed: 0          SP         WT
0           1  104.185353  28.762059
1           2  105.461264  30.466833
2           3  105.461264  30.193597
3           4  113.461264  30.632114
4           5  104.461264  29.889149

In [3]:
df.SP.skew()

1.6114501961773586

In [4]:
df.WT.skew()

-0.6147533255357768

In [5]:
df.skew()

Unnamed: 0    0.000000
SP            1.611450
WT           -0.614753
dtype: float64

# skewness value for speed and distance is 1.611450 and -0.614753

In [6]:
df.SP.kurt()

2.9773289437871835

In [7]:
df.WT.kurt()

0.9502914910300326

In [8]:
df.kurt()

Unnamed: 0   -1.200000
SP            2.977329
WT            0.950291
dtype: float64

In [ ]:
Kurtosis value for speed and distance is 2.977329 and 0.950291